### ISSUE TITLE
Investigate VA.gov Experience Standard 11.22 - "Alert and status messages aren't announced without receiving focus."

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#43515]
## User Story
As a Governance team member, I want to have clear and well documented standards to follow so that I can easily ascertain what standards are in violation when reviewing a VFS team's product.

As a VFS team member, I want to easily understand what issue my teams product is being flagged with so that I can implement the correct solution as easily and efficiently as possible.

Assignee:
Peer Reviewer:

## Description
In the Staging Review for Patient Check-in, @briandeconinck flagged an issue related to a status alert NOT receiving focus. The issue was flagged using the "Other" category because there was not a VA.gov Experience Standard that described the issue accurately enough. In this scenario, it would make sense for the alert to receive focus.

Upon review, @shiragoodman noticed that there is a VA.gov Experience Standard (11.22) that specifies the problematic behavior, as, "Alert and status messages aren't announced without receiving focus." Per @briandeconinck, "there's a use case where we want it to get focus and a use case where we don't." The VA.gov Experience Standard only covers one of the use cases.

The purpose of this ticket is to research VA.gov Experience Standard 11.22 and identify if it needs modification or if a new standard should be created to address the gaps.

## Impacted Artifacts
[Platform Website: VA.gov Experience Standard](https://vfs.atlassian.net/wiki/spaces/CC/pages/1683980311/VA.gov+experience+standards)
[Drafting Template Source](https://docs.google.com/spreadsheets/d/1IkIX7CgbFneFvlia1r9mYD8QU7uML5HWILkM7qsxizs/edit#gid=333168944)

## Tasks
- [ ] Determine if Experience Standard 11.22 should cover both use cases
  - [ ] Draft modified Experience Standard if so
  - [ ] Draft new Experience Standard if not
- [ ] Share updated/new Experience Standard with the team
- [ ] Update Platform Website: VA.gov Experience Standard list
- [ ] Update Drafting Template Source file

## Peer Review
- [ ] Ticket reassigned to peer reviewer when ready for peer review
- [ ] Peer Reviewer: Completes review and validates user story/Acceptance Criteria are met

## Acceptance Criteria
- [ ] Governance team members are able to use updated or new standards for applicable use cases in reviews
- [ ] VFS teams understand the Experience Standard and can reference any available WCAG resources

## How to prepare this issue
- [ ] Attach story to appropriate Iniative or Quarterly Tech-Debt Epic.
- [ ] Add User Story, description, tasks, and Acceptance Criteria
- [ ] Add any [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization).
- [ ] Once all above items are checked, add the 'Ready' label.
- [ ] Ticket has been discussed as a team and sized.
- [ ] When brought into a sprint, move to the Planned pipeline.
- [ ] When brought into a sprint, add assignee and specify Peer Reviewer.

**Comment by shiragoodman** [comment]
@briandeconinck FYI. Please feel free to expand on the description if I did not do a good job explaining the problem.

**Comment by briandeconinck** [comment]
No longer applicable. New experience standards should handle both focus/alert situations.
